File: Python Word examples/background/setBackgroundImage.py
import spirecloudword
from spirecloudword.configuration import Configuration as WordConfiguration

appId = "your id"
appKey = "your key"

configuration = WordConfiguration(appId, appKey)
api = spirecloudword.api.background_api.BackgroundApi(configuration)
file_name = "BackgroundSample.docx"
image_path = "input/Logo.png"
destFilePath = "/output/SetBackgroundImage.docx"
result = api.set_backgroud_image(file_name, image_path, folder="input", dest_file_path=destFilePath)

File: JavaScript Word Examples/BackgroundApiDemo/setBackgroundImage.js
(function (){
    var appId = "your id";
    var appKey = "your key";
    var baseUrl = "https://api.e-iceblue.cn";

    var Spirecloudword = require('../../src/index');
    var apiClient = new Spirecloudword.ApiClient(appId, appKey,baseUrl);
    var instance = new Spirecloudword.BackgroundApi(apiClient);
    
    var name = "AddBackground.docx";
    var destFilePath = 'output/Background/SetBackgroundImage.docx';
    var imagePath = "background/Hydrangeas.jpg";
     var opt = {folder: 'background'};
    var response = instance.setBackgroundImage(name, imagePath, destFilePath, opt,
        function (error, data, response) {        
            if (error) throw error;
        });
})();